app/src/main/res/values/dimens.xml

```diff
@@ -96,6 +96,23 @@
     <!-- Home Fragment -->
     <dimen name="home_fragment_top_toolbar_header_margin">60dp</dimen>
     <dimen name="home_collection_elevation">5dp</dimen>
+    <dimen name="home_recently_saved_padding_top">40dp</dimen>
+    <dimen name="home_recently_saved_padding_bottom">16dp</dimen>
+    <dimen name="home_recently_saved_padding_start">16dp</dimen>
+    <dimen name="home_recently_saved_padding_end">16dp</dimen>
+    <dimen name="home_show_all_padding_top">16dp</dimen>
+    <dimen name="home_show_all_padding_bottom">16dp</dimen>
```

Comment:
I am kinda curious by seeing all these new `dimen` variables when we should be adding them in general. I have seen us add variables when it needs to be used programmatically through our kotlin files and when it's reusable across layouts and components. However, looking at how small this `dimens.xml` file is, I am wondering how we should plan to go about this here and in the future?

My immediate thought is that this might be useful if we were responsive and implementing for different device sizes, but historically, we have not done this.

Reply:
Not sure. I'll keep these here for now.